modules/serverless-tech-preview-features.adoc

```diff
@@ -15,15 +15,15 @@ The following table provides information about which {ServerlessProductName} fea
 |====
 |Feature |1.29|1.30|1.31
 
-|Using Service Mesh to isolate network traffic with OpenShift Serverless
+|Installing and using {ServerlessProductShortName} on {sno}
 |-
 |-
-|TP
+|GA
 
-|Golang functions
+|Using {SMProductShortName} to isolate network traffic with {ServerlessProductShortName}
 |-
 |-
-|TP
+|GA
```

Comment:
```suggestion
|TP
```

If the "|Using {SMProductShortName} to isolate network traffic with {ServerlessProductShortName}" refers to 
"{ServerlessProductName} multi-tenancy with {SMProductShortName} is now available as a Technology Preview (TP) feature.",  it should be marked as TP